testproject/Assets/Tests/Manual/Scripts/ManualNetworkVariableTest.cs

```diff
@@ -106,34 +106,40 @@ private void ValueChanged(int before, int after)
             {
                 // compute the delta in tick between client and server,
                 // as seen from the client, when it receives a value not from itself
-                if (m_TestVar.LocalTick != NetworkTickSystem.NoTick)
-                {
-                    int delta = m_TestVar.LocalTick - m_TestVar.RemoteTick;
-                    m_Count++;
-
-                    if (!m_Valid)
-                    {
-                        m_Valid = true;
-                        m_MinDelta = delta;
-                        m_MaxDelta = delta;
-                        m_LastRemoteTick = m_TestVar.RemoteTick;
-                    }
-                    else
-                    {
-                        m_MinDelta = Math.Min(delta, m_MinDelta);
-                        m_MaxDelta = Math.Max(delta, m_MaxDelta);
-
-                        // tick should not go backward until wrap around (which should be a long time)
-                        if (m_TestVar.RemoteTick == m_LastRemoteTick)
-                        {
-                            m_Problems += "Same remote tick receive twice\n";
-                        }
-                        else if (m_TestVar.RemoteTick < m_LastRemoteTick)
-                        {
-                            m_Problems += "Ticks went backward\n";
-                        }
-                    }
-                }
+
```

Comment:
@jeffreyrainy sorry to dump on this part...I'm hoping and assuming you're going to tell me that this test would be invalid anyway as we replace with snapshot.  I hesitated to try and re-engineer it for that reason